scripts/script_utils.py

```diff
@@ -16,15 +16,34 @@
 
 from __future__ import print_function
 
+import json
 import os
+import re
+import sys
 import subprocess
 
+# Mimic six here, but avoid any dependencies outside
+# the standard library.
```

Comment:
You don't have to make it a separate package, but if you make script_utils its own package within this repo you can solve all of these issues for now.